src/stores/database.js

```diff
@@ -115,6 +116,14 @@ class Database {
     await umzug.up();
   }
 
+  async generateSchemaHash() {
+    const migrations = await this._db.collection('migrations').find({}).toArray();
+    const migrationNames = migrations.map(migration => migration.name).join();
```

Comment:
Would it be good to sort the names just to make extra extra sure?

Reply:
I considered the order but thing is it's the exact same code that places these records, right?
But ok, it would not hurt.

Reply:
Well, it should in theory be like that, but you know theory and practice sometimes diverge :))

Reply:
Done!